mpp/build.gradle.kts

```diff
@@ -178,12 +181,18 @@ tasks.register("publishComposeJbExtendedIconsToMavenLocal", ComposePublishingTas
     iconsPublications()
 }
 
+// TODO deprecated, kept for CI compatibility, remove after Compose Multiplatform 1.9 is released
```

Comment:
after 1.8?

Reply:
This should work, changed.

Even it should work after I merge the corresponding CI change, but better to keep in case of reverts